app.py

```diff
@@ -194,18 +192,18 @@ def user_leaderboard_view(login):
 
 @app.template_filter('min')
 def min_filter(l):
-  return min(l)
+  return min(l) if l else 0
 
 @app.template_filter('sum')
 def sum_filter(l):
-  return sum(l)
+  return sum(l) if l else 0
 
 @app.template_filter('lang_nice')
 def lang_nice_filter(s):
   return extensions.get_language_from_extension(s)
 
 if __name__ == '__main__':
   if dev:
-    app.run(host='0.0.0.0', port=5000, debug=True)
+    app.run(host='0.0.0.0', port=5000, debug=False)
```

Comment:
i actually found that when i ran `heroku local` with `debug=True`, it swallowed uncaught exceptions. with `debug=False`, it emitted them to stdout (or stderr).

not sure why, and i don't feel strongly. i'm happy to revert this or fix that another way if you know one!